File: 1) Arrays/easy/1480. Running Sum of 1d Array/Solution.java
class Solution {
  static int[] runningSum(int[] nums) {
      for (int i=1; i<nums.length; i++) {
          nums[i] = nums[i] + nums[i-1];
      }
      
      for (int el: nums) {
        System.out.print(el + " - ");
      }
      return nums;
  }

  public static void main(String[] args) {
    runningSum(new int[]{1,2,3,4});
  }
}

File: 1) Arrays/easy/1480. Running Sum of 1d Array/Solution.js
const nums1 = [1,2,3,4];
const nums2 = [1,1,1,1,1];
const nums3 = [3,1,2,10,1];

function runningSum (arr) {
  for (let i=1; i<arr.length; i++) {
    arr[i] = arr[i] + arr[i-1];
  };

  return arr;
}

console.log(runningSum(nums1));
console.log(runningSum(nums2));
console.log(runningSum(nums3));